peloton/update_project.py

```diff
@@ -0,0 +1,1371 @@
+#!/usr/bin/env python3
+"""
+Script for populating the Peloton GitHub project with Issues and Discussions.
+
+https://github.com/orgs/SciTools/projects/13?pane=info
+
+Achieved by running GraphQL queries and mutations against GitHub - the
+recommended way to work with GitHub data.
+https://docs.github.com/en/graphql
+
+The GraphQL syntax is difficult to work with directly, so we use sgqlc to
+provide a pythonic interface. `github_schema.py` helps enumerate what
+operations are possible. (For advance use, consider using a GraphQL IDE or IDE
+plugin, and parse the schema directly - code completion and syntax highlighting
+works well, and sgqlc-codegen can also convert GraphQL into Python code).
+
+"""
+
+import abc
+from argparse import ArgumentParser
+from dataclasses import dataclass, asdict, fields
+from datetime import date, datetime, timedelta
+from enum import StrEnum
+import logging
+from string import printable
+from time import sleep
+from sys import stdout
+
+import numpy as np
+import pandas as pd
+
+from sgqlc.endpoint.http import HTTPEndpoint
+import sgqlc.types
+import sgqlc.types.relay
+import sgqlc.operation
+import github_schema
+
+github_schema = github_schema
+github_schema_root = github_schema.github_schema
+
+
+GITHUB_QUERY_CONDITIONS = (
+    "org:SciTools org:SciTools-incubator org:SciTools-classroom "
+    "-repo:SciTools/cartopy "
```

Comment:
@trexfeathers What does the `-repo` mean here?

Reply:
2b4dd92